salt/_modules/metalk8s_checks.py

```diff
@@ -16,6 +16,47 @@ def __virtual__():
     return __virtualname__
 
 
+def bootstrap_file():
```

Comment:
I would prefer to check using the value from the pillar directly so that you will have the defaults & co and do not have to manage the potential various version of bootstrap file

Reply:
I can't access it in the pillar at this early step of execution,
I can maybe move this forward right before the bootstrap states ?

Reply:
Hmm are you sure ?
This step runs after the "configure local salt minion" so the pillar should work here already to me

Reply:
you were right about this, I made the move to using pillar